model/pipeline/src/main/proto/beam_runner_api.proto

```diff
@@ -245,6 +245,25 @@ message StandardPTransforms {
     COMBINE_PGBKCV = 0 [(beam_urn) = "beam:transform:combine_pgbkcv:v1"];
     COMBINE_MERGE_ACCUMULATORS = 1 [(beam_urn) = "beam:transform:combine_merge_accumulators:v1"];
     COMBINE_EXTRACT_OUTPUTS = 2 [(beam_urn) = "beam:transform:combine_extract_outputs:v1"];
+
+    // Represents the lifted part of a Combine.perKey() operation. Caches and
```

Comment:
Better to refer to the name and point to the doc then the java code and a vague description.

Reply:
Done